src/export/StructureDefinitionExporter.ts

```diff
@@ -178,6 +175,107 @@ export class StructureDefinitionExporter implements Fishable {
     }
   }
 
+  /**
+   * Handles a ContainsRule that is on an extension path, appropriately exporting it as a reference to a standalone
+   * extension or an inline extension.
+   * @param {Profile|Extension} fshDefinition - the FSH Definition the rule is on
+   * @param {ContainsRule} rule - the ContainsRule that is on an extension element
+   * @param {StructureDefinition} structDef - the StructDef of the resulting profile or element
+   * @param {ElementDefinition} element - the element to apply the rule to
+   */
+  private handleExtensionContainsRule(
+    fshDefinition: Profile | Extension,
+    rule: ContainsRule,
+    structDef: StructureDefinition,
+    element: ElementDefinition
+  ) {
+    if (!element.slicing) {
+      element.sliceIt('value', 'url');
+    }
+    rule.items.forEach(item => {
+      if (item.type) {
+        const extension = this.fishForMetadata(item.type, Type.Extension);
+        if (extension == null) {
+          logger.error(
+            `Cannot create ${item.name} extension; unable to locate extension definition for: ${item.type}.`,
+            rule.sourceInfo
+          );
+          return;
```

Comment:
As I see you went with the `return` in the `forEach`. I hope I haven't swayed you out of your preferred ways.

Reply:
Ha.  Yeah, I guess maybe I came at it with a more open mind than I previously would have.  That's not a bad thing!